jdi-light/src/main/java/com/epam/jdi/light/asserts/generic/table/IGridAssert.java

```diff
@@ -0,0 +1,225 @@
+        for (int i = 1; i < column.size(); i++)
+            if (!isSorted(column.get(i), column.get(i-1), true, false))
+                jdiAssert("Table is not by descending at "+i+" row", Matchers.is(""));
+        return (A)this;
+    }
+    @JDIAction("Assert that '{name}' is sorted by '{0}' in ascending order")
+    public A sortedNumByAsc(String columnName) {
+        List<String> column = map(grid().webColumn(columnName).values(), el -> el.replaceAll("[^0-9.,]", ""));
+        for (int i = 1; i < column.size(); i++)
+            if (!isSorted(column.get(i-1), column.get(i), true, false))
+                jdiAssert("Table is not by ascending at "+i+" row", Matchers.is(""));
+        return (A)this;
+    }
+    @JDIAction("Assert that '{name}' is sorted by '{0}' in descending order")
+    public A sortedNumByDesc(String columnName) {
+        List<String> column = map(grid().webColumn(columnName).values(), el -> el.replaceAll("[^0-9.,+\\-]", ""));
+        for (int i = 1; i < column.size(); i++)
+            if (!isSorted(column.get(i), column.get(i-1), true, false))
+                jdiAssert("Table is not by descending at "+i+" row", Matchers.is(""));
+        return (A)this;
+    }
+
+     public Compare exact(int count) {
+         return new Compare(count, (A) this, EXACT);
+     }
+     public Compare atLeast(int count) {
+         return new Compare(count, (A) this, ATLEAST);
+     }
+     public Compare no() {
+         return exact(0);
+     }
+     public Compare all() {
+         return new Compare((A) this);
+     }
+     public Compare onlyOne() {
+         return exact(1);
+     }
+
+     public enum CompareType {
+         EXACT("exactly"), ATLEAST("at least"), ALL("all");
+         public String text;
+         CompareType(String text) { this.text = text; }
+     }
+
+     public class Compare implements JAssert {
+         public int count;
+         public String name;
+         public String printText;
+         A dtAssert;
+         CompareType compareType;
+         public JDIBase base() { return this.base(); }
+
+         private Compare(A dtAssert) {
+             this.dtAssert = dtAssert;
+             this.compareType = ALL;
+             this.printText = "all rows";
+             this.name = dtAssert.name;
+         }
+         private Compare(int count, A dtAssert, CompareType compareType) {
+             this.count = count;
+             this.dtAssert = dtAssert;
+             this.compareType = compareType;
+             this.printText = format("has %s %s rows that", compareType.text, count);
+             this.name = dtAssert.name;
+         }
+
+         /**
+          * Check that the table has rows that meet expected condition
+          * @param condition to compare
+          * @return DataTableAssert
+          */
+         @JDIAction("Assert that '{name}' {printText} meet expected condition")
+         public A rows(JFunc1<D, Boolean> condition) {
+             switch (compareType) {
```

Comment:
Codacy found an issue: [Switch statements should have a default label](https://app.codacy.com/gh/jdi-testing/jdi-light/pullRequest?prid=6421085)